adi/rx_tx.py

```diff
@@ -210,7 +224,8 @@ def __rx_non_complex(self):
 
             for c in range(stride):
                 raw = x[c::stride]
-                sig.append(raw * rx_scale[c] + rx_offset[c])
+                # when scale is used, values are in mV
+                sig.append((raw * rx_scale[c] + rx_offset[c]) / 1000.0)
```

Comment:
This change should be in the device specific class not the core.

Reply:
The whole SI part(lines 204-228) or just the mV part (225-228) ?

Reply:
228 the divide by 1000